lib/components/player/stronzflix_player_controller.dart

```diff
@@ -48,15 +48,35 @@ abstract class StronzflixPlayerController {
     bool get isCompleted;
     StronzflixPlayerStream get stream;
 
+    bool get isEpisode => this._watchable is Episode;
+
     String get title {
         if (this._watchable is Film)
             return this._watchable.name;
-        else if (this._watchable is Episode)
-            return '${this._watchable.season.series.name} - ${this._watchable.name}';
+        else if (this._watchable is Episode) {
+            final String seasonNoPadded = this._watchable.season.seasonNo.toString().padLeft(2, '0');
+            final String episodeNoPadded = this._watchable.episodeNo.toString().padLeft(2, '0');
+            return 'S${seasonNoPadded}:E${episodeNoPadded} ${this._watchable.season.series.name} ${this._watchable.name}';
+        }
         else
             throw Exception('Unknown watchable type');
     }
 
+    String? get seriesTitle {
+        if (this._watchable is! Episode) return null;
+
+        return this._watchable.season.series.name;
+    }
+
+    String? get episodeTitle {
+        if (this._watchable is! Episode) return null;
+
+        final String seasonNoPadded = this._watchable.season.seasonNo.toString().padLeft(2, '0');
+        final String episodeNoPadded = this._watchable.episodeNo.toString().padLeft(2, '0');
+
+        return 'S${seasonNoPadded}:E${episodeNoPadded} ${this._watchable.name}';
+    }
+
```

Comment:
If any of this still remains in this file and is not moved into the correct place, it should be a PR to the video player repository cited above and not this one.